mobile/lib/ui/viewer/file/file_bottom_bar.dart

```diff
@@ -39,10 +42,27 @@ class FileBottomBar extends StatefulWidget {
 
 class FileBottomBarState extends State<FileBottomBar> {
   final GlobalKey shareButtonKey = GlobalKey();
+  bool _isFileSwipeLocked = false;
+  @override
+  void initState() {
+    super.initState();
+    Bus.instance.on<FileSwipeLockEvent>().listen((event) {
+      setState(() {
+        _isFileSwipeLocked = event.shouldSwipeLock;
+      });
+    });
+  }
+
+  @override
+  void dispose() {
+    super.dispose();
+  }
 
   @override
   Widget build(BuildContext context) {
-    return _getBottomBar();
+    return _isFileSwipeLocked
+        ? IgnorePointer(child: Container())
```

Comment:
Instead of `IgnorePointer(child: Container())`, if the goal is to not show anything here, can return a `SizedBox.shrink()` instead.